RELEASE.md

```diff
@@ -117,6 +120,29 @@ Promotion should occur in two steps:
 
 **NOTE**: The promotion of wheels to PyPI can only be done once so take caution when attempting to promote wheels to PyPI, (see https://github.com/pypa/warehouse/issues/726 for a discussion on potential draft releases within PyPI)
 
+# Doing a Patch Release
+
+A patch release is a maintenance release of PyTorch that includes fixes for regressions found in a previous minor release.
+
+## Things to consider for a patch release
+
+1. Does the regression break core functionality?
+2. Does the regression break first party domain libraries that depend on locked versions of `pytorch`?
+    * [pytorch/vision](https://github.com/pytorch/vision)
+    * [pytorch/audio](https://github.com/pytorch/audio)
+    * [pytorch/text](https://github.com/pytorch/text)
+3. Is there not a viable workaround?
+
+## Patch release process
+
+1. Tag issues / pull requests with `patch release triage`
+    * ![image](https://user-images.githubusercontent.com/1700823/131175874-1526d098-d69c-48e4-ad75-c4afa1099f62.png)
```

Comment:
what is the purpose of this?  Please describe.  What is the difference between this and putting the issue into the milestone?

Reply:
Adding a secondary label for this would notify everyone that it's up for discussion specifically for a patch release.

Another option is to instead do this by just adding it to the milestone and adding the `triage review` label, but I thought adding another label would make things a bit more clear.

Reply:
that makes sense.  I would expect this document to basically answer two questions:
1.  How do we decide whether there will be a patch release?
2.  How do we organize things into the patch release

It's not really clear currently where one ends and the other begins.

Reply:
Okay I've updated this with a more in-depth guide of how patch releases should go down